### PR TITLE
docs(error/dupes): describe your change...

### PR DESCRIPTION
**Pull request by RakeshYadvanshi**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
doc update

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?**

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information**:

I think this help's , new users of java script and angular ,understanding the way javascript word and create error to angular repeater.

### CODE REVIEW

**Comment by RakeshYadvanshi** [comment]
I signed it!

File: docs/content/error/ngRepeat/dupes.ngdoc [hunk 1]
```diff
@@ -20,3 +20,16 @@ The example above can be resolved by using `track by $index`, which will cause t
...
+In Detail, if same object is used to add in the array, that create this error. Reason is in javascript a object never dies. That's why if same name object is used to insert in array
+than error comes.
+
+```javascript
+var customers=[];
+var obj={name:"rk"};
+customers.push(obj); // work fine
+customers.push(obj); // give same error if used in repeater
+obj={name:"rk1"};
+customers.push(obj); // give same error if used in repeater
```
Comment (gkalpak) [inline docs/content/error/ngRepeat/dupes.ngdoc]: No, it doesn't.

**Comment by gkalpak** [comment]
Sorry, but I feel that explaining such basic JavaScript concepts is outside the scope of the Angular documentation. Adding too many non-Angular specific details clutters the docs and is more distracting than helpful for the majority of users.

Thx for taking the time to submit the PR though :+1:
Maybe next time :smiley:

File: docs/content/error/ngRepeat/dupes.ngdoc (hunk 1 above)
Comment (RakeshYadvanshi) [inline docs/content/error/ngRepeat/dupes.ngdoc]: i don't thinks so.
Reason being i have faced this error.

**Comment by RakeshYadvanshi** [comment]
its Ok

**Pull request closed without merging**